### ISSUE TITLE
Transaction details action menu button label is incorrect

### ISSUE THREAD

**Issue by Jinksi** [Automattic/woocommerce-payments#7964]
### Describe the bug

The transaction details action menu button is labelled as `Translation actions`, where it should have a correct label such as `Transaction actions`.

<img width="1177" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/b402ddf3-3c60-41b2-b36f-255a2faddd3a">

### To Reproduce

1. Go to a transaction details screen
2. Mouse over the three-dot menu icon in the top-right.
3. Notice the popover text showing `Translation actions`

**Comment by kalessil** [comment]
@Jinksi: the label is intended - the menu is intended for adding more actions (which ones is not decided yet).

**Comment by KarlisJ** [comment]
@kalessil @Jinksi The truth is in between both of you. The label should be "**Transaction** actions" but mistakenly it's "**Translation** actions".

I'll create PR today.

**Comment by Jinksi** [comment]
> The label should be "Transaction actions" but mistakenly it's "Translation actions".

Thanks, I was wrongly assuming this was just for refunds! I'll update the description to mention `Transaction actions`.
